### PR TITLE
Move test classes to test main folder;

### PR DESCRIPTION
**Pull request by mat1th**
## Summary
This pr does move test files to the test folder of the project.

## Screenshots
NA

## Link to pull request in Documentation repository
-

## Any other notes
-

### CODE REVIEW

File: Sources/App/Scenes/SettingsSceneDelegate.swift [hunk 1]
```diff
@@ -150,7 +151,8 @@ extension SettingsSceneDelegate: NSToolbarDelegate {
         }
     }
 
-    @objc private func selectItem(_ item: NSToolbarItem) {
```
Comment (bgoncal) [inline Sources/App/Scenes/SettingsSceneDelegate.swift]: Did the linter update this?

File: Sources/App/Utilities/UIImage+scaledToSize.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Shared
+import UIKit
+
+extension UIImage {
```
Comment (bgoncal) [inline Sources/App/Utilities/UIImage+scaledToSize.swift]: Can you make a folder for extensions? I saw we have  a few inside Utilities

**Review by bgoncal — CHANGES_REQUESTED** [review]
Minor comments, but looking good! Thanks for helping out

File: Sources/App/Scenes/SettingsSceneDelegate.swift (hunk 1 above)
Comment (mat1th) [reply Sources/App/Scenes/SettingsSceneDelegate.swift]: Yes, can revert it.

**Review by bgoncal — APPROVED** [review]

**Pull request merged**